tests/neteventebpfext/neteventebpfext_unit/netevent_ebpfext_unit.cpp

```diff
@@ -40,18 +40,17 @@ void
 _dump_event(uint8_t event_type, const char* event_descr, void* data, size_t size)
 {
     if ((event_type == NOTIFY_EVENT_TYPE_NETEVENT_DROP || event_type == NOTIFY_EVENT_TYPE_NETEVENT_LOG) &&
-        size == sizeof(netevent_message_t)) {
+        size == sizeof(netevent_payload_t)) {
 
         // Cast the event and print its details
-        netevent_message_t* demo_event = reinterpret_cast<netevent_message_t*>(data);
-        std::cout << "\rNetwork event [" << demo_event->event_counter << "]: {"
-                  << "src: " << (int)demo_event->source_ip.octet1 << "." << (int)demo_event->source_ip.octet2 << "."
-                  << (int)demo_event->source_ip.octet3 << "." << (int)demo_event->source_ip.octet4 << ":"
-                  << demo_event->source_port << ", "
-                  << "dst: " << (int)demo_event->destination_ip.octet1 << "." << (int)demo_event->destination_ip.octet2
-                  << "." << (int)demo_event->destination_ip.octet3 << "." << (int)demo_event->destination_ip.octet4
-                  << ":" << demo_event->destination_port << ", "
-                  << "reason: " << (int)demo_event->reason;
+        netevent_payload_t* demo_payload = reinterpret_cast<netevent_payload_t*>(data);
```

Comment:
nit: suggest minor renaming to "test_payload" or similar (this isn't really a demo, it's a test)